src/mockServiceWorker.js

```diff
@@ -8,20 +8,23 @@
 /* tslint:disable */
 
 const INTEGRITY_CHECKSUM = '<INTEGRITY_CHECKSUM>'
-const bannerStyle = 'color:orangered;font-weight:bold;'
 const bypassHeaderName = 'x-msw-bypass'
 
+let clients = {}
```

Comment:
Code declared in the global scope of the worker can be executed multiple times during the worker's lifecycle. This declaration should be moved to the worker's instance (`self`) to guarantee that the worker doesn't clear up its clients map unexpectedly.

Reply:
I've tried setting it within `activate`, or keeping the declaration only in the global scope, yet both options yield broken worker logic. I'm leaving this as it is until further notice.